components/eventing-controller/pkg/backend/jetstreamv2/utils_test.go

```diff
@@ -0,0 +1,517 @@
+package jetstreamv2
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/kyma-project/kyma/components/eventing-controller/pkg/env"
+	"github.com/nats-io/nats.go"
+
+	kymalogger "github.com/kyma-project/kyma/common/logging/logger"
+	eventingv1alpha2 "github.com/kyma-project/kyma/components/eventing-controller/api/v1alpha2"
+	"github.com/kyma-project/kyma/components/eventing-controller/logger"
+	"github.com/kyma-project/kyma/components/eventing-controller/pkg/backend/cleaner"
+	evtestingv2 "github.com/kyma-project/kyma/components/eventing-controller/testing/v2"
+	"github.com/stretchr/testify/require"
+)
+
+// maxJetStreamConsumerNameLength is the maximum preferred length for the JetStream consumer names
+// as per https://docs.nats.io/running-a-nats-service/nats_admin/jetstream_admin/naming
+const (
+	subName                        = "subName"
+	subNamespace                   = "subNamespace"
+	maxJetStreamConsumerNameLength = 32
```

Comment:
Shouldn't `maxJetStreamConsumerNameLength` be defined in `types.go` and reused in both code and test? 
Also a nit, would const defined in _test.go file be visible/accessible in whole pkg?

Reply:
Actually this is just used for tests to confirm that the md5 32bit hash actually generates 32 characters. So I left it here